packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -3,43 +3,31 @@
  */
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Button } from '@wordpress/components';
-import { LEFT, RIGHT, UP, DOWN, BACKSPACE, ENTER } from '@wordpress/keycodes';
+import { Button, Notice } from '@wordpress/components';
 import { keyboardReturn } from '@wordpress/icons';
-
```

Comment:
The line should have been kept, as is conventional for separating import groupings (admittedly could be clearer in [the documentation](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#imports)).

Reply:
Rebase error.

Reply:
Fixed.